app/views/widgets/home_page/_chapman_family.html

```diff
@@ -1,6 +1,6 @@
 <section id="chapmanFamily" class="section light-bg stories">
   <div class="maxWidth">
-    <h2 class="heading" skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">One Chapman Family. Many #ChapmanStories.</h2>
+    <h2 class="heading" skrollable skrollable-after data-bottom="opacity:0;" data-center="opacity:1;">One Chapman Family. Many #ChapmanStories.</h2>
```

Comment:
I think the line should be this:

     <h2 class="heading skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">